python/ml4ir/applications/ranking/tests/test_base.py

```diff
@@ -126,7 +126,7 @@ def get_ranking_model(
             learning_rate=self.args.learning_rate,
             learning_rate_decay=self.args.learning_rate_decay,
             learning_rate_decay_steps=self.args.learning_rate_decay_steps,
-            gradient_clip_value=self.args.gradient_clip_value,
+            clipvalue=self.args.gradient_clip_value,
```

Comment:
this needs to be clipvalue now, because this is the tf variable name, and its name + value will be passed directly there. I know you prefer 'clip_value', but this is TF not me.

Reply:
Haha. No worries.